host-controller/src/main/java/org/jboss/as/domain/controller/resources/DomainRootDefinition.java

```diff
@@ -361,7 +361,7 @@ public void initialize(ManagementResourceRegistration resourceRegistration) {
         @Override
         public void validateParameter(String parameterName, ModelNode plan) throws OperationFailedException {
             if(plan == null) {
-                throw new OperationFailedException(DomainControllerLogger.ROOT_LOGGER.nullVar("plan").getLocalizedMessage());
+                throw new OperationFailedException(DomainControllerLogger.ROOT_LOGGER.nullVar("plan"));
```

Comment:
This should have been an assert in the first place -- https://issues.jboss.org/browse/WFCORE-1769. I'll fix that after I merge this.